Package.swift

```diff
@@ -18,13 +18,14 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/ddddxxx/Semver", from: "0.2.1")
```

Comment:
I chose this package because it supported swift 5.0, which is a fairly old version that we are supporting.